tests/events.middleware.get.test.ts

```diff
@@ -176,9 +178,9 @@ describe('events.middleware.get', () => {
     const channel = 'channel';
 
     it('parses after to be int within [0, MAX_SAFE_INTEGER]', () => {
-      const t = (desiredAfter, expected) => {
-        const actual = parseGetParams({clientId, channel, after: desiredAfter});
-        expect(actual.after).to.equal(expected);
+      const t = (desiredAfter: number | undefined | string | {}, expected: number) => {
+        const actual = parseGetParams({ clientId, channel, after: desiredAfter });
+        expect((actual as any)['after']).to.equal(expected);
```

Comment:
`actual.after` should work here. Maybe `parseGetParams` doesn't return the appropriate type?